src/organizer_en.ts

```diff
@@ -110,22 +110,22 @@
         <translation type="unfinished"></translation>
     </message>
```

Comment:
Ah this got committed as well, do you think you can revert it to avoid annoying conflicts when pulling locally with umbrella?